src/Symfony/Bridge/Doctrine/Types/UlidType.php

```diff
@@ -15,9 +15,11 @@
 
 final class UlidType extends AbstractUidType
 {
+    public const NAME = 'ulid';
```

Comment:
maybe use the new constant also here https://github.com/symfony/symfony/blob/6.2/src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterUidTypePass.php#L38

during registration of the type?

Reply:
Yep, and probably also [here](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php#L86)? What about tests? If the constant is used in tests, I guess there should be tests asserting `'uuid' === UuidType::NAME` and `'ulid' === UlidType::NAME`, too?

Reply:
would make sense, but maybe wait for feedback from Symfony team before trying to find all relevant locations 😄